docs/README.md

```diff
@@ -2,6 +2,11 @@
 
 Documentation for the PISP demo server
 
+## Design
+
+There are two main scenarios that are handled by the PISP demo server:
+- [Transfer](./design/transfer.md)
+
```

Comment:
?

Reply:
The other point should be:
```
- [Linking](./design/linking.md)
```

However, it is currently written only on the other draft PR, do you think it's better to put that one here as well? Or perhaps I can change the phrasing to:

```
Here are the scenarios that are handled by the PISP Demo server:
```

Reply:
Updated the phrasing to the second option